documentation/explainers/synthetic_tokens.md

```diff
@@ -0,0 +1,112 @@
+- tokens that track the future usage of DeFi projects (e.g. assets locked in Uniswap)
+- tokens that track the number of downloads of a Chrome extension (e.g. Metamask)
+- tokens that track the success of trade ideas on r/WallStreetBets 
+
+## Priceless Synthetic Tokens
+“Priceless” synthetic tokens are synthetic tokens that are securely collateralized without an on-chain price feed. These tokens are designed with mechanisms to incentivize token sponsors (those who create synthetic tokens) to properly collateralize their positions. These mechanisms include a liquidation and dispute process that allows token holders to be rewarded for identifying improperly collateralized token sponsor positions. The dispute process relies on an oracle, the UMA DVM, to settle disputes regarding liquidations. 
+
+For more details on how a “priceless” framework can be applied more generally to other types of financial contract templates on the blockchain, check out this blog post. <!-- TODO: add link to blog post -->
+
+## Launching a New Synthetic Token
+To launch a new type of synthetic token for which an existing market does not yet exist, that synthetic token’s smart contract must first be parameterized and deployed. Anyone can parameterize and deploy this contract. 
+
+This smart contract can be deployed using UMA’s synthetic token contract template and defining key parameters. Some parameters to highlight are:
+- token’s price identifier (the price feed this token should track)
+- token expiration timestamp
+- token collateralization ratio (e.g. a synthetic token must have collateral worth at least 120% of the price indentifier’s current value) 
+
+This [tutorial](../tutorials/creating_synthetic_tokens_from_truffle_console.md) will show you how to parameterize and deploy the smart contract for a new synthetic token from the command line.
+
+## Creating Synthetic Tokens From an Existing Contract
+After the contract is deployed, anyone can interact with the contract to create synthetic tokens. These actors are called “token sponsors”, since they sponsor the creation of synthetic tokens. 
+
+<p align="center">
+  <img src="st_create_contract.png" width="500">
+</p>
+
+Token sponsors deposit collateral into the contract to collateralize synthetic tokens, which they can then withdraw and trade with others. The first token sponsor to create synthetic tokens is able to immediately withdraw synthetic tokens from the contract. Any following token sponsors who wish to create synthetic tokens can do so, but must collateralize their positions by at least as much as the system’s global collateralization ratio. This prevents token sponsor positions from being immediately liquidated. <!-- TODO: add link to glossary for “global collateralization ratio” --> 
```

Comment:
"This prevents token sponsor positions from being immediately liquidated"... Allison pointed out to me that this isn't technically correct. They can still dispute (there's no rule against that), they would just be wrong. Can we rephrase?

Reply:
I agree that this isn't technically correct, especially because it doesn't describe the mechanism by which it is effected, but I believe it conveys the point that we are trying to make - that collateralizing to the GCR provides some assurance that your position will not be immediately liquidated. 

What do you think of adding this paragraph as a separate footnote that we can link to? 

The global collateralization ratio (GCR) is the average collateralization ratio among all token sponsor positions that have not yet been liquidated nor have any outstanding withdrawal requests. Requiring new token sponsors to collateralize their positions by as much as the GCR provides some assurances that so long as those token sponsors below the GCR have not yet been liquidated, those above the GCR need not be at risk of liquidation.

Reply:
I think your paragraph is great and should live here (in addition to having an (even more detailed?) version in the glossary. I don't think it should be a footnote.

Can you add this paragraph and rework modestly to make it flow? I think we should also change the ""This prevents token sponsor positions from being immediately liquidated" to something like "The insures that all newly created tokens sponsor positions are correctly collateralized upon initialization".

Reply:
Yes, I'll do this in the next commit.